app/views/widgets/addresslookup.html

```diff
@@ -52,9 +52,9 @@
                 document.getElementById('newPostCode').value = postcode;
                 document.getElementById('country').value = 'United Kingdom';
 
-                document.getElementById('details-panel').setAttribute('open', null);
-                document.getElementById('details-summary').setAttribute('aria-expanded', true);
-                document.getElementById('details-content-0').setAttribute('aria-hidden', false);
+                document.getElementById('details-panel').setAttribute('open', true);
```

Comment:
it was `null` before

Reply:
Doesn't really make any difference, just nicer to read :)

Reply:
there is a difference between `null` and `true` tho?

Reply:
Not really, either will add the `open` attribute without a value, ie. `<details class="govuk-details" id="details-panel" role="group" open>`